### ISSUE TITLE
Both bundle.js and bundle.min.js included in dist

### ISSUE THREAD

**Issue by dceddia** [terser-js/terser#404]
**Bug report or Feature request?**

Bug, probably?

**Version (complete output of `terser -V` or specific git commit)**

terser 3.17.0

**Complete CLI command or `minify()` options used**

```
$ node_modules/terser/bin/uglifyjs -V
terser 3.17.0

$ ls node_modules/terser/dist
bundle.js         bundle.js.map     bundle.min.js     bundle.min.js.map

$ du -shc node_modules/terser/dist/*
780K	node_modules/terser/dist/bundle.js
1.4M	node_modules/terser/dist/bundle.js.map
332K	node_modules/terser/dist/bundle.min.js
1.1M	node_modules/terser/dist/bundle.min.js.map
3.6M	total
```

I might be wrong about this, but does terser need to include both the `bundle.js` *and* the minified `bundle.min.js` along with both of the .map files? Looking at `package.json`, the "main" file is `dist/bundle.min.js` and it seems like `bundle.js` is unused. If that's true, and the bundle+map could be removed, it'd save 2.2MB, which is like 60% of the package size 😄

**Comment by fabiosantoscode** [comment]
I very much agree with you. We just assumed bundle.js was going to be useful for something, but it wasn't.

**Comment by fabiosantoscode** [comment]
I'm going to take this chance to get rid of the separate test bundle and provide exports starting with `_` because everyone understands those are not part of the API.

**Comment by fabiosantoscode** [comment]
Has been fixed and as of 4.1.4 the bundle is now smaller. I'm sorry for the extra disk space that has been used.
